src/app/onion/learning-object-builder/components/builder-navbar/builder-navbar.component.ts

```diff
@@ -6,16 +6,22 @@ import { filter, takeUntil } from 'rxjs/operators';
 import { Subject } from 'rxjs';
 import { Router, ActivatedRoute } from '@angular/router';
 import { ToastrOvenService } from 'app/shared/modules/toaster/notification.service';
-import { CollectionService, Collection } from 'app/core/collection-module/collections.service';
+import {
+  CollectionService,
+  Collection,
+} from 'app/core/collection-module/collections.service';
 import { LearningObject } from '@entity';
-import { HistoryService, HistorySnapshot } from 'app/core/client-module/history.service';
+import {
+  HistoryService,
```

Comment:
how's the history service look did that not need any changes? thought that'd be what holds the specific learning object's details page to navigate back to